constants/ProjectThemes.ts

```diff
@@ -0,0 +1,52 @@
+export interface IProjectTheme {
+  id: string
+  image: string
+  name: string
+}
+
+const ProjectTheme: IProjectTheme[] = [
+  {
+    id: '1',
+    image:
+      'https://cdn.dribbble.com/users/3281732/screenshots/11192830/media/7690704fa8f0566d572a085637dd1eee.jpg?compress=1&resize=1200x1200',
```

Comment:
**Quick question:**  
When would it make sense to pack the assets, and when to fetch from cdn instead?